### PR TITLE
kubuntuMainline

### PR DESCRIPTION
**Pull request by lemmyg**
Kubuntu 23.04 version.
required pkgs for KDE
plymouth-theme-kubuntu-logo 
plymouth-theme-kubuntu-text
kde-plasma-desktop 
plasma-discover 
kubuntu-wallpapers 
kubuntu-desktop
kubuntu-settings-desktop 
sddm-theme-breeze 
ubiquity-slideshow-kubuntu 
ubuntu-drivers-common

remove:
sddm-theme-debian-maui to use sddm-theme-breeze

### CODE REVIEW

File: files/chroot_build.sh [hunk 1]
```diff
@@ -75,18 +76,25 @@ echo >&2 "===]> Info: Install window manager... "
...
-  ubuntu-gnome-wallpapers \
-  snapd
+  kde-plasma-desktop \
+  kubuntu-settings-desktop \
+  plasma-discover \
+  kubuntu-wallpapers \
+  kde-plasma-desktop \
+  sddm-theme-breeze \
+  snapd \
+  firefox
```
Comment (AdityaGarg8) [inline files/chroot_build.sh]: You cannot install snap packages in chroot, so this shall fail to install. So probably remove firefox.

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

**Comment by AdityaGarg8** [comment]
CI also needs to be edited to make a release and push changes to kubuntuMainline-16,4

File: files/chroot_build.sh (hunk 1 above)
Comment (lemmyg) [reply files/chroot_build.sh]: ok

**Comment by lemmyg** [comment]
> CI also needs to be edited to make a release and push changes to kubuntuMainline-16,4

do I need to create a specific branch for kubuntuMainline-16,4 or I just need to update the .github/workflows/CI.yml file?

Im not familiar with CI

**Comment by AdityaGarg8** [comment]
> > CI also needs to be edited to make a release and push changes to kubuntuMainline-16,4
> 
> do I need to create a specific branch for kubuntuMainline-16,4 or I just need to update the .github/workflows/CI.yml file?
> 
> Im not familiar with CI

If that's the case, then I'll manage the CI. Can you just do the required changes for kubuntu in mainline-16,4 branch as well and make a kubuntuMainline-16,4 branch?

**Comment by lemmyg** [comment]
Sure

**Comment by lemmyg** [comment]
> > > CI also needs to be edited to make a release and push changes to kubuntuMainline-16,4
> > 
> > 
> > do I need to create a specific branch for kubuntuMainline-16,4 or I just need to update the .github/workflows/CI.yml file?
> > Im not familiar with CI
> 
> If that's the case, then I'll manage the CI. Can you just do the required changes for kubuntu in mainline-16,4 branch as well and make a kubuntuMainline-16,4 branch?

done

File: files/chroot_build.sh [hunk 2]
```diff
@@ -66,27 +66,34 @@ apt-get install -y -qq -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="
   grub2 \
   nautilus-admin
 
-curl -L https://github.com/t2linux/T2-Ubuntu-Kernel/releases/download/vKVER-PREL/linux-headers-KVER-${ALTERNATIVE}_KVER-PREL_amd64.deb > /tmp/headers.deb
-curl -L https://github.com/t2linux/T2-Ubuntu-Kernel/releases/download/vKVER-PREL/linux-image-KVER-${ALTERNATIVE}_KVER-PREL_amd64.deb > /tmp/image.deb
+curl -L https://github.com/t2linux/T2-Ubuntu-Kernel/releases/download/v6.2.12-1/linux-headers-6.2.12-${ALTERNATIVE}_6.2.12-1_amd64.deb > /tmp/headers.deb
```
Comment (AdityaGarg8) [inline files/chroot_build.sh]: Why this change?

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: files/chroot_build.sh (hunk 2 above)
Comment (lemmyg) [reply files/chroot_build.sh]: probably a mistake. Ill revert them
